### PR TITLE
Add C++ large message performance tests to PRs and master

### PR DESCRIPTION
**Pull request by vjpai**

### CODE REVIEW

**Comment by vjpai** [comment]
Test failures: #9609 (fixed by #9807) and #9668

**Comment by vjpai** [comment]
Jenkins, test this please

**Comment by vjpai** [comment]
Rebased from master, no change in diff.

**Comment by vjpai** [comment]
Note that this is not related to #10380 because this is for unconstrained tests, not ping-pong

**Comment by vjpai** [comment]
Test failures: #10388 #10199 #10390

**Comment by vjpai** [comment]
Mac backlog = Maclog. Not relevant here, so merging.

File: tools/run_tests/performance/scenario_config.py [hunk 1]
```diff
@@ -215,6 +215,29 @@ def scenarios(self):
           categories=smoketest_categories+[SCALABLE])
 
       yield _ping_pong_scenario(
+          'cpp_generic_async_streaming_qps_1channel_1MBmsg_%s' % secstr,
```
Comment (jtattermusch) [inline tools/run_tests/performance/scenario_config.py]: nit: I think other perf tests use "secstr_msgsize" naming pattern and not  "msgsize_secstr". Also, they're not using the "msg" suffix.

**Pull request merged**